app/interactors/foreman_netbox/sync_host/sync_device/sync_site/organizer.rb

```diff
@@ -8,7 +8,7 @@ class Organizer
           include ::Interactor::Organizer
 
           around do |interactor|
-            interactor.call if context.host.location
+            interactor.call if context.host.location && !Setting[:netbox_skip_site_update]
```

Comment:
This setting is about modifying existing Sites but should not break the creation of new ones. One more setting is probably needed, like `netbox_create_sites`.

Reply:
Went with `netbox_upsert_sites` since if disabled then neither create nor update are wanted.